app/src/main/java/com/github/se/polyfit/ui/utils/Authentication.kt

```diff
@@ -44,6 +45,16 @@ class AuthenticationCloud @Inject constructor(private val context: Context) : Au
     Log.i("LoginScreen", "response: $response")
     if (result.resultCode == Activity.RESULT_OK) {
       Log.i("LoginScreen", "User signed in")
+      // to get google acount infos
+      val account = GoogleSignIn.getLastSignedInAccount(context)
+      User.currentUser =
```

Comment:
- We need to figure out a way to pass this data to other screens, as I'm encountering the same issue in PR #58. Let's get together and discuss a possible implementation.